### PR TITLE
add CFG service

### PR DESCRIPTION
**Pull request by Scalaidoscope**
Add a service that extracts the control flow graphs (CFG) of the binaries via an open-source tool called 'nucleus'.

P.S.: This is my first PR, so please be gentle! 😉

### CODE REVIEW

File: src/main/scala/org/holmesprocessing/totem/services/cfg/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# CFG specific options
+###
+
+# Get Go CFG dependencies
+RUN apk add --no-cache \
+		bash \
+		build-base \
+		binutils \
+		binutils-dev \
+		capstone \
```
Comment (boddumanohar) [inline src/main/scala/org/holmesprocessing/totem/services/cfg/Dockerfile]: When trying to build Dockerfile, this line shows an error. Are you sure about the name `capstone`?

Comment (Scalaidoscope) [reply src/main/scala/org/holmesprocessing/totem/services/cfg/Dockerfile]: Oh yeah, it's in the edge/testing repo - I updated the Dockerfile accordingly. Does it build now?

Comment (boddumanohar) [reply src/main/scala/org/holmesprocessing/totem/services/cfg/Dockerfile]: Still some errors and warning from the Dockerfile. Make sure that it builds completely on your machine.

**Comment by cynexit** [comment]
Hey there! :)
A CFG service is a very nice idea, let's make this happen! I tried to build and run the container of your service and got this:
```
cp3 ../totem/services/cfg ✔ docker run -p 8080 -v /tmp:/tmp:ro --name="totem_cfg_server" totem_cfg
2017/05/03 14:51:22 Unable to locate nucleus binary, is nucleus installed? exec: "nucleus": executable file not found in $PATH
```
Can you check this error out?

**Pull request merged**